src/migrations/tasks/09.add-last-updated-data.ts

```diff
@@ -0,0 +1,47 @@
+import { baseConfig } from "../../models/Config";
+import { D2Api } from "../../types/d2-api";
+import { addAttributeValueToObj } from "../../models/Attributes";
+import { Debug, Migration } from "../types";
+import { post } from "./common";
+
+class AddLastUpdatedDataMigration {
+    constructor(private api: D2Api, private debug: Debug) {}
+
+    async execute() {
+        await this.updateOrgUnits();
+    }
+
+    async updateOrgUnits() {
+        const { organisationUnits, attributes } = await this.api.metadata
+            .get({
+                organisationUnits: {
+                    fields: { $owner: true },
+                    filter: { level: { eq: baseConfig.orgUnits.levelForCountries.toString() } },
+                },
+                attributes: {
+                    fields: { id: true, code: true },
+                    filter: { code: { eq: baseConfig.attributes.lastUpdatedData } },
+                },
+            })
+            .getData();
+
+        const orgUnitAttribute = attributes[0];
+
+        const orgUnitsUpdated = organisationUnits.map(organisationUnit =>
+            addAttributeValueToObj(organisationUnit, {
+                attribute: orgUnitAttribute,
+                value: "",
+            })
+        );
+
+        await post(this.api, this.debug, { organisationUnits: orgUnitsUpdated });
```

Comment:
No need to update the organisation units, the attribute will be set from now on when users enter data.

Instead, the idea of the migration is to create the attribute itself. Note that this feature will be deployed to instances where the attribute does not exist, so the app will start with an error ("Attribute XYZ not found"). We don't want to create it manually every time. That's why we create a migration, so it's automatic.